src/Controller/CRUDController.php

```diff
@@ -1464,7 +1464,7 @@ final protected function checkParentChildAssociation(Request $request, object $o
 
         $parentAssociationMapping = $this->admin->getParentAssociationMapping();
         if (null === $parentAssociationMapping) {
-            throw new \RuntimeException('The admin has no parent association mapping.');
```

Comment:
This exception was introduced in 4.x because there was a TODO in 3.x, but there was no deprecation.

I got at least 2 people with issues with this exception.